src/onefetch/cli.rs

```diff
@@ -243,7 +255,13 @@ impl Cli {
         let no_color_palette = matches.is_present("no-color-palette");
         let print_languages = matches.is_present("languages");
         let print_package_managers = matches.is_present("package-managers");
-        let print_in_json_format = matches.is_present("json");
+        let iso_time = matches.is_present("isotime");
+
+        let output = if let Some(output) = matches.value_of("output") {
+            Some(SerializationFormat::from_str(&output).unwrap())
+        } else {
+            None
+        };
```

Comment:
Just another way to write the same thing 🙂 
```suggestion
        let output = matches.value_of("output").map(SerializationFormat::from_str).transpose().unwrap();
```